components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam4_resus_mom_soag_mosaic_tag.in

```diff
@@ -0,0 +1,300 @@
+Comments
```

Comment:
@mingxuanwupnnl, Why is this file needed for the coupling between MOSAIC and MAM5?

Reply:
@hwangacme it is only for chemUCI-MOSAIC, I did not put chem_mech.in there for chemUCI-MOSAIC previously.